File: src/main/java/com/example/studentmanagementsystem/StudentManagementSystemApplication.java
package com.example.studentmanagementsystem;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/** @author Hingbong */
@SpringBootApplication
@MapperScan("com.example.studentmanagementsystem.mappers")
public class StudentManagementSystemApplication {

  public static void main(String... args) {
    SpringApplication.run(StudentManagementSystemApplication.class, args);
  }
}

File: src/main/kotlin/com/example/studentmanagementsystem/StudentManagementSystemApplication.kt
package com.example.studentmanagementsystem

import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@MapperScan("com.example.studentmanagementsystem.mappers")
class StudentManagementSystemApplication

fun main(args: Array<String>) {
    runApplication<StudentManagementSystemApplication>(*args)
}
